src/plots/plots.js

```diff
@@ -1332,6 +1332,7 @@ plots.purge = function(gd) {
     delete gd._transitionData;
     delete gd._transitioning;
     delete gd._initialAutoSize;
+    delete gd._transitioningWithDuration;
```

Comment:
This was just one minor cleanup tweak I noticed along the way.

Reply:
Good catch :ok_hand: